entrypoint.sh

```diff
@@ -6,9 +6,17 @@ set -e
 # If not will, the container will fail and restart.
 python3 manage.py shell -c "import django; django.db.connection.ensure_connection();"
 
-# On PROD, we run migrations at startup unless explicitly disabled.
+# On PROD, we run migrations at startup unless explicitly disabled. If disabled,
+# this commands must be run manually
 if [ "$ENV" == "PROD" ] && [ "${DISABLE_MIGRATION_SCRIPT_ON_PRODUCTION}" != "true" ]; then
-    scripts/migrate.sh
+    python3 manage.py migrate
+    python3 manage.py update_integrator_permissions
```

Comment:
Just to be sure, it's on purpose to run `update_integrator_permissions` only with `migrate`, and in all cases run `compilemessages` and `collectstatic`

Reply:
As discussed with @monodo, we run `update_integrator_permissions` and `compilemessages` every time, regardless of theenvironment.